src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,23 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table
```

Comment:
Not just a table annotation itself, but with the name of the DB table that represents this entity
```suggestion
@Table(name = "actors")
```